pkg/shares/non_interactive_defaults.go

```diff
@@ -34,35 +34,35 @@ func BlobSharesUsedNonInteractiveDefaults(cursor, squareSize int, blobShareLens
 	start := cursor
 	indexes = make([]uint32, len(blobShareLens))
 	for i, blobLen := range blobShareLens {
-		cursor, _ = NextMultipleOfBlobMinSquareSize(cursor, blobLen, squareSize)
+		cursor, _ = NextShareIndex(cursor, blobLen, squareSize)
 		indexes[i] = uint32(cursor)
 		cursor += blobLen
 	}
 	return cursor - start, indexes
 }
 
-// NextMultipleOfBlobMinSquareSize determines the next index in a square that is
-// a multiple of the blob's minimum square size. This function returns false if
-// the entire the blob cannot fit on the given row. Assumes that all args are
-// non negative, and that squareSize is a power of two.
+// NextShareIndex determines the next index in a square that can be used. It
+// follows the non-interactive default rules defined in ADR013. This function
+// returns false if the entire the blob cannot fit on the given row. Assumes
+// that all args are non negative, and that squareSize is a power of two.
 // https://github.com/celestiaorg/celestia-specs/blob/master/src/rationale/message_block_layout.md#non-interactive-default-rules
-// https://github.com/celestiaorg/celestia-app/blob/1b80b94a62c8c292f569e2fc576e26299985681a/docs/architecture/adr-009-non-interactive-default-rules-for-reduced-padding.md
-func NextMultipleOfBlobMinSquareSize(cursor, blobLen, squareSize int) (index int, fitsInRow bool) {
+// https://github.com/celestiaorg/celestia-app/blob/0334749a9e9b989fa0a42b7f011f4a79af8f61aa/docs/architecture/adr-013-non-interactive-default-rules-for-zero-padding.md
```

Comment:
Ideally, this links to the spec instead of ADR013 as this ADR builds upon quite a bit of existing context and won't be as complete as an explanation for how the square is constructed than the spec should be.